mne/viz/_brain/tests/test.ipynb

```diff
@@ -39,16 +42,40 @@
     "initial_time = 0.13\n",
     "mne.viz.set_3d_backend('notebook')\n",
     "brain_class = mne.viz.get_brain_class()\n",
-    "for interactive_state in (False, True):\n",
-    "    plt.interactive(interactive_state)\n",
+    "\n",
+    "\n",
+    "@contextmanager\n",
+    "def interactive(on):\n",
+    "    old = plt.isinteractive()\n",
+    "    plt.interactive(on)\n",
+    "    try:\n",
+    "        yield\n",
+    "    finally:\n",
+    "        plt.interactive(old)\n",
+    "\n",
+    "with interactive(False):\n",
```

Comment:
I'm not sure how critical the iteration over interactive levels was so I killed it to save time now that this isn't marked `slowtest`.